t/167-atomic-shdict.t

```diff
@@ -0,0 +1,658 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+use Test::Nginx::Socket::Lua;
+
+#worker_connections(1014);
+#master_process_enabled(1);
+#log_level('warn');
+
+#repeat_each(2);
```

Comment:
It looks like this test case can be executed repeatedly, maybe we could uncomment this line